File: xkcd/src/debug/java/xyz/jienan/xkcd/DebugUtils.java
package xyz.jienan.xkcd;

import android.app.Application;
import android.content.Context;

import com.gu.toolargetool.TooLargeTool;

import timber.log.Timber;

public class DebugUtils {
    static boolean init() {
        if (BuildConfig.DEBUG) {
            Timber.plant(new Timber.DebugTree());
        }
        return true;
    }

    static void debugDB(Context context) {
        if (BuildConfig.DEBUG) {
            TooLargeTool.startLogging((Application) context);
        }
    }
}

File: xkcd/src/debug/java/xyz/jienan/xkcd/DebugUtils.kt
package xyz.jienan.xkcd

import android.app.Application
import android.content.Context
import com.gu.toolargetool.TooLargeTool.startLogging
import timber.log.Timber

object DebugUtils {
    fun init(): Boolean {
        if (BuildConfig.DEBUG) {
            Timber.plant(Timber.DebugTree())
        }
        return true
    }

    fun debugDB(context: Context?) {
        if (BuildConfig.DEBUG) {
            startLogging((context as Application?)!!)
        }
    }
}